packages/subgraph/schema.graphql

```diff
@@ -462,152 +452,356 @@ type StandardCallback @entity {
   transfers: [TokenTransfer!]! @derivedFrom(field: "dao")
+
+  " DAO balances for all kinds of tokens (refer to `Interfaces/TokenBalance` in this documentation site for more information). "
   balances: [TokenBalance!]! @derivedFrom(field: "dao")
+
+  " The permissions given to a DAO to govern its actions (refer to `Objects/Permission` in this documentation site for more information). "
   permissions: [Permission!]! @derivedFrom(field: "dao")
+
+  " The proposals to execute actions that a DAO has had. "
   proposals: [IProposal!]! @derivedFrom(field: "dao")
+
+  " The address of the trusted forwarder required for meta transactions, if added while creating the DAO. "
   trustedForwarder: Bytes
+
+  " Deprecated field. "
   signatureValidator: Bytes
+
+  " Registered callback functions supported by the DAO (refer to `Objects/StandardCallback` in this documentation site for more information). "
   standardCallbacks: [StandardCallback!]! @derivedFrom(field: "dao")
 
+  " The plugins that had been installed/uninstalled to the DAO. "
   plugins: [PluginInstallation!]! @derivedFrom(field: "dao")
 }
 
-# ACL
+#ACL
 
 # Cannot be immutable because a permission can be revoked and granted in the same TX.
 # This results in an error in the subgraph if the entity is immutable.
 # No need to store granted as we can delete permission when revoked
+
+"""
+[Aragon permission system](https://devs.aragon.org/docs/osx/how-it-works/core/permissions/), permissions between contracts and wallets allow
+our DAOs (and `PluginRepo`s) to manage and govern its actions. A permission is the hash of a struct which specifies an address `who` being
+allowed to call certain functions on a contract address `where`.
+"""
 type Permission @entity {
-  id: ID! # emitting contract Address (DAO or PluginRepo) + permissionId + where + who
+  " Concatenation of the emitting contract Address (DAO or PluginRepo), the `permissionId`, `where` and `who`. Joined by '_'. "
+  id: ID!
+
+  " The address of the target contract to grant `who` permission to. "
   where: Bytes!
+
+  " Bytes32, the Keccak256 hash function of the ID for the permission. Normally the Keccak256 of the permission name (string). "
   permissionId: Bytes!
+
+  " The address (EOA or contract) to which the permission has been granted. "
   who: Bytes!
+
+  " The address of the context in which the permission has been granted. "
   actor: Bytes!
+
+  " The address of the contract containing the conditions which authorize a call by another address. This contract must inherit from `PermissionCondition` and implement the `IPermissionCondition` interface. "
   condition: Bytes
 
-  # The following attributes is used to create
+  # The following attributes are used to create
   # one-to-many between DAO or PluginRepo to Permission
+
+  " The DAO that grants the permission (DAO uses `PermissionManager` SC `grant` function). "
```

Comment:
I'd avoid `SC` as per above - it's implied when you're already reading about a subgraph